services/api/service.go

```diff
@@ -49,6 +49,7 @@ import (
 )
 
 const (
+	maxMessageSize        = 15 * 1024 * 1024 // 15 MiB
```

Comment:
might be useful to make configurable through env var

Reply:
`maxPayloadSize` would be a better name. otherwise it's confusing, like "which message"